### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 <!--
     Don't forget!
-    
+
+    - Sign your commits and our DCO:  https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
```
Comment (roidelapluie) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: We should not require GPG signing of git commits. That sets the bar too high.

It's better to have a relevant DCO link. Also, it is not "our" DCO.

**Review by roidelapluie — COMMENTED** [review]
Thanks for this. We could also link to CONTRIBUTING.md.

**Review by bwplotka — APPROVED** [review]
Thanks! As, @roidelapluie mentioned, do we want to mention Contributing doc? (:

**Comment by roidelapluie** [comment]
let's move forward with this, thanks!

**Pull request merged**